pinc/gradual.inc

```diff
@@ -126,4 +131,36 @@ function maybe_output_new_proofer_message()
     }
 }
 
+function maybe_output_new_proofer_project_message()
+{
+    global $code_url, $ELR_round;
+
+    // Help direct new proofers to projects to proof.
+    $pagesproofed = get_pages_proofed_maybe_simulated();
+    if($pagesproofed < 100)
+    {
+        echo "<div class='callout'>";
+        echo "<div class='calloutheader'>";
+        echo _("Welcome New Proofreader!");
+        echo "</div>";
+
+        echo "<p>";
+        echo _("You are a click away from proofreading a page. Take a moment to scroll down and read the Project Comments which contain instructions specific to this project. Once you're ready to start proofreading, click the \"Start Proofreading\" link.");
```

Comment:
I'm always wary of referencing other strings in a string in case one gets updated and this one is left behind, leaving a confusing user experience. I'm particularly referring to "Start Proofreading".

Reply:
This is a fair fear. In this case I'm less worried about the string being changed in one place and not the other and more worried that translators will translate the string in two different ways -- which is a very real concern. I will pull this out into a separate string but keep it duplicated as we have no way of making this the same string without a random global variable which seems like the wrong approach.

Reply:
I agree about it being a legitimate concern, but didn't like the idea of forcing it to be a global variable. Pulling it out as a separate string sounds like a good idea. That way, it'll match the string that's already translated (assuming the capitalization matches), and won't need a second translation.

Reply:
See the last commit for this change.